openedx/adg/lms/webinars/tests/test_tasks.py

```diff
@@ -0,0 +1,28 @@
+"""
+All tests for webinar tasks
+"""
+from datetime import datetime, timedelta
+
+import pytest
+
+from openedx.adg.lms.webinars.tasks import task_reschedule_webinar_reminders
+
+from .factories import WebinarFactory, WebinarRegistrationFactory
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'new_start_time, expected_call_count', [
+        (datetime.now() + timedelta(days=1), 1),
+        (datetime.now() + timedelta(days=8), 2),
+    ]
+)
+def test_task_send_mandrill_email_successfully(new_start_time, expected_call_count, mocker):
+    """
+    Tests if reschedule_webinar_reminders is called properly when the start_time is updated.
+    """
+    mock_reschedule = mocker.patch('openedx.adg.lms.webinars.tasks.reschedule_webinar_reminders')
+    webinar = WebinarFactory()
+    WebinarRegistrationFactory(webinar=webinar)
+    task_reschedule_webinar_reminders(webinar.id, new_start_time.strftime('%m/%d/%Y, %H:%M:%S'))
+    assert expected_call_count == mock_reschedule.call_count
```

Comment:
can we add some spacing similar to the other tests, it would make this code more readable.